mlir/include/mlir-c/Pass.h

```diff
@@ -76,7 +76,15 @@ mlirPassManagerRunOnOp(MlirPassManager passManager, MlirOperation op);
 
 /// Enable mlir-print-ir-after-all.
```

Comment:
nit: change this to be more accurate? or just remove it? I personally never can see the point of comments like this lol.

Reply:
Thanks for the suggestion and reviewing the MR! I would change the description to make it more accurate